wagtail/core/models.py

```diff
@@ -74,17 +73,6 @@ def _extract_field_data(source, exclude_fields=None):
                     values = parental_field.all()
                     if values:
                         data_dict[field.name] = values
```

Comment:
Now that we're no longer building `m2m_dict` here, the comment above this block (`# Otherwise add them to the m2m dict to be set after saving`) needs updating.